src/ziggurat/streams.clj

```diff
@@ -92,6 +97,22 @@
         additional-tags  {:topic_name topic-entity-name}]
     (.transform stream-builder (transformer-supplier metric-namespace oldest-processed-message-in-s additional-tags) (into-array [(.name (store-supplier-builder))]))))
 
+(defn- traced-handler-fn [handler-fn channels message-payload topic-entity]
+  (let [parent-ctx (TracingKafkaUtils/extractSpanContext (:headers message-payload) tracer)
+        span (-> tracer
+                 (.buildSpan "Message-Handler")
+                 (.asChildOf parent-ctx)
+                 (.withTag (.getKey Tags/SPAN_KIND) Tags/SPAN_KIND_CONSUMER)
+                 (.withTag (.getKey Tags/COMPONENT) "lambda")
```

Comment:
what does this tag signify?? shouldn't it be `ziggurat` instead of `lambda`?

Reply:
this is just to classify the consumers. yes it should be ziggurat.